### DISCUSSION TITLE
Api call for actions

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#358] [Q&A] [answered]
Hi @mgineer85 ,

is there an api call to take a  picture in the same way as the button trigger it ?
Including count down an dthe whole process

**Comment by mgineer85** [comment, marked as answer]
Of course there is. And it is well documented and includes interactive elements to your specific booth :)

--> start your booth and go to http://localhost:8000/api/doc
The frontend only uses this API 🍕

Example:
- choose the image action
- click TRY
- use index=0 for the first configuration for the specific image-action
- click execute.
- watch the booth in another browser executing the countdown, capture the picture and present it :)
- like it ... hopefully 🤠

![image](https://github.com/user-attachments/assets/e819e483-44c9-46cb-a795-0e52a4ea3544)

**Comment by Striker62** [comment]
H, thanks .I tried this but did not work. I had not the correct syntax.
